### PR TITLE
Yard CRUD

### LINKED ISSUE #23: Create a new yard
POST /api/v1/yards

### LINKED ISSUE #24: Update a yard
PUT /api/v1/yards/:yard_id

### LINKED ISSUE #25: Delete a yard
DELETE /api/v1/yards/:yard_id

### PR DESCRIPTION
**Pull request by jbranham1**
PR #40 needs to be merged before this one!
Added create, update, delete functionality for yard

Close #23 
Close #24 
Close #25

### CODE REVIEW

**Review by abreaux26 — APPROVED** [review]

File: app/controllers/api/v1/yards_controller.rb [hunk 1]
```diff
@@ -10,11 +10,43 @@ def show
...
 
+  def create
+    yard = Yard.create!(yard_params)
+    render json: YardSerializer.new(yard), status: :created
+  end
+
+  def update
+    yard = Yard.find(params[:id])
+    yard.update!(yard_params)
+    render json: YardSerializer.new(yard)
```
Comment (Gvieve) [inline app/controllers/api/v1/yards_controller.rb:21]: How are you handling sad path/edge cases for update? For example what if a required attribute is not included or is an empty string?

**Review by Gvieve — COMMENTED** [review]
I left a comment on the update method.

File: app/controllers/api/v1/yards_controller.rb (hunk 1 above)
Comment (jbranham1) [reply app/controllers/api/v1/yards_controller.rb:21]: @Gvieve On the application controller there is a rescue for an invalid record!

Comment (Gvieve) [reply app/controllers/api/v1/yards_controller.rb:21]: Ahh I overlooked that. I see it now. Thanks!

**Review by Gvieve — APPROVED** [review]
Thanks for the quick response. Looks great!

**Pull request merged**